org-cyf-itp/content/structuring-data/sprints/1/day-plan/index.md

```diff
@@ -5,35 +5,40 @@ emoji= '🧑🏾‍🤝‍🧑🏾'
 menu_level = ['sprint']
 weight = 3
 [[blocks]]
-name="Energiser"
-src="blocks/energiser"
-[[blocks]]
 name="Morning orientation"
 src="blocks/morning-orientation"
-time=15
+time=10
+[[blocks]]
+name="Energiser: Popcorn"
+src="energisers/popcorn-screen-share"
+[[blocks]]
+name="Javascript Test Your Understanding"
+src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/js1-wk1-eval"
+time=65
 [[blocks]]
-name="Workshop"
-src="blocks/workshop"
-time="140"
-  [[blocks.nested.blocks]]
-    name="Reporting Bugs [PD] (60 Mins)"
-    src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/reporting-bugs"
-    time="0"
-  [[blocks.nested.blocks]]
-    name="Playing Computer [Tech] (60 Mins)"
-    src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/playing-computer"
-    time="0"
-  [[blocks.nested.blocks]]
-    name="Javascript Test Your Understanding [Tech] (30 Mins)"
-    src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/js1-wk1-eval"
-    time="0"
+name="Morning break"
+src="blocks/morning-break"
+[[blocks]]
+name="Another Asking Questions Workshop"
+src="https://cyf-pd.netlify.app/blocks/asking-good-technical-questions"
+time=70
```

Comment:
This workshop mentions prep, but we haven't assigned the prep in the prep or backlog?

Reply:
Combined into https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/11